crates/astria-account-monitor/src/config.rs

```diff
@@ -44,12 +44,31 @@ pub struct Config {
     pub metrics_http_listener_addr: String,
 }
 
-#[derive(Debug, Clone, Hash, Serialize)]
+#[derive(Debug, Clone, Hash, Serialize, PartialEq, Eq)]
 pub struct Asset {
```

Comment:
Do you still need this type at all? Can't you just do the below and forego `Asset` entirely?

```rust
struct SequencerAssetsToMonitor(Vec<Denom>)`
```